zerver/lib/webhooks/git.py

```diff
@@ -75,6 +75,15 @@
 TAG_WITHOUT_URL_TEMPLATE = "{tag_name}"
 
 RELEASE_MESSAGE_TEMPLATE = "{user_name} {action} release [{release_name}]({url}) for tag {tagname}."
+RELEASE_MESSAGE_TEMPLATE_WITHOUT_URL = (
+    "{user_name} {action} release {release_name} for tag {tagname}."
+)
```

Comment:
Let's just remove this as it has not been used as of now.

Reply:
Yes, removed!